maintenance_request_ifs_check/views/maintenance_views.xml

```diff
@@ -0,0 +1,80 @@
+        </field>
+    </record>
+
+    <!-- New tree view: MaintenanceRequest - added ifs_checked -->
+    <record id="hr_equipment_request_view_tree" model="ir.ui.view">
+        <field name="name">equipment.request.tree</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='stage_id']" position="after">
+                <field name="ifs_checked" invisible="1"/>
+            </xpath>
+            <!-- decoration-bf - shows the line in BOLD
+                decoration-it - shows the line in ITALICS
+                decoration-danger - shows the line in LIGHT RED
+                decoration-info - shows the line in LIGHT BLUE
+                decoration-muted - shows the line in LIGHT GRAY
+                decoration-primary - shows the line in LIGHT PURPLE
+                decoration-success - shows the line in LIGHT GREEN
+                decoration-warning - shows the line in LIGHT BROWN -->
+            <xpath expr="//tree" position="attributes">
+                <attribute name='decoration-success'>ifs_checked</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <!-- New form view: MaintenanceRequest - added new IFS checks -->
+    <record id="hr_equipment_request_view_form" model="ir.ui.view">
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_form"/>
+        <field name="priority" eval="100"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='description']" position="replace">
+                <notebook>
+                    <page string="Description">
+                        <field name="description" placeholder="Internal Note ......."/>
+                    </page>
+                    <page string="IFS checks">
+                        <group>
+                            <group>
+                                <field name="ifs_check_repaired_correctly"/>
+                                <field name="ifs_check_free_strange_elements"/>
+                            </group>
+                            <group>
+                                <field name="ifs_check_collected_location"/>
+                                <field name="ifs_check_cleaning_area"/>
+                            </group>
+                        </group>
+                    </page>
+                </notebook>
+            </xpath>
+        </field>
+    </record>
+
+    <!-- New search view: MaintenanceRequest - added ifs_checked -->
+    <record id="hr_equipment_request_view_search" model="ir.ui.view">
+        <field name="name">equipment.request.search</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_search"/>
+        <field name="sequence" eval="18"/>
```

Comment:
I don't know the purpose of `sequence` attribute

Reply:
I do not know either. I think I copied it from another view. I put the same sequence in my other PR [12.0][ADD] maintenance_request_time_performance#1... 
I should put 16 on both, right?